sys/parsoid.js

```diff
@@ -224,7 +230,7 @@ class ParsoidService {
     }
 
     getFallbackBucketURI(rp, format, tid) {
-        return this.getStashBucketURI(rp, format, tid);
+        return this.getOldStashBucketURI(rp, format, tid);
```

Comment:
I think this is a relic from the transition to the complicated storage semantics. AFAICT, we don't need it any more, right?

Reply:
Still need. this is for non-latest revision edits.